examples/flutter_gallery/lib/demo/material/chip_demo.dart

```diff
@@ -217,16 +216,16 @@ class _ChipDemoState extends State<ChipDemo> {
 
     final List<Widget> inputChips = _tools.map<Widget>((String name) {
       return InputChip(
-          key: ValueKey<String>(name),
-          avatar: CircleAvatar(
-            backgroundImage: _nameToAvatar(name),
-          ),
-          label: Text(_capitalize(name)),
-          onDeleted: () {
-            setState(() {
-              _removeTool(name);
-            });
+        key: ValueKey<String>(name),
+        avatar: CircleAvatar(
+          backgroundImage: _nameToAvatar(name),
+        ),
+        label: Text(_capitalize(name)),
+        onDeleted: () {
+          setState(() {
+            _removeTool(name);
           });
+        });
```

Comment:
The automatic trailing comma fixup technology seems to have failed here. This should be formatted like:

```
    final List<Widget> inputChips = _tools.map<Widget>((String name) {
      return InputChip(
        key: ValueKey<String>(name),
        // ...
        onDeleted: () {
          setState(() {
            _removeTool(name);
          });
        },
      );
    });
```